aws-ebs-csi-driver/values.yaml

```diff
@@ -29,6 +29,11 @@ sidecars:
     repository: quay.io/k8scsi/csi-node-driver-registrar
     tag: "v1.1.0"
 
+snapshotController:
```

Comment:
should this be considered a sidecar like the other images? imo yes

Reply:
My bad, it is not a sidecar, and we aren't supposed to bei nstalling it. REF: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/635